web/src/compose_closed_ui.js

```diff
@@ -53,32 +53,42 @@ export function get_recipient_label(message) {
     return "";
 }
 
-function update_reply_button_state(disable = false, title) {
+function update_reply_button_state(disable = false) {
     $(".compose_reply_button").attr("disabled", disable);
-    if (!title) {
-        const title_text = $t({defaultMessage: "Reply to selected message"});
-        title = title_text + " (r)";
+    if (disable) {
+        $("#compose_buttons > .reply_button_container").attr(
+            "data-tooltip-template-id",
+            "compose_reply_button_disabled_tooltip_template",
+        );
+        return;
+    }
+
+    if (narrow_state.is_message_feed_visible()) {
+        $("#compose_buttons > .reply_button_container").attr(
+            "data-tooltip-template-id",
+            "compose_reply_message_button_tooltip_template",
+        );
+    } else {
+        $("#compose_buttons > .reply_button_container").attr(
+            "data-tooltip-template-id",
+            "compose_reply_selected_topic_button_tooltip_template",
+        );
```

Comment:
I changed this to have more rational template IDs -- I don't know why `left_bar` is all over this code path but let's avoid adding too many more references.

Also split a "Recent conversations" variant since the tooltip seemed wrong there.